ci-operator/config/openshift-kni/eco-ci-cd/openshift-kni-eco-ci-cd-main__far-edge-vran-dast-4.16.yaml

```diff
@@ -0,0 +1,41 @@
+base_images:
+  base:
+    name: "4.16"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    name: ci
+    namespace: openshift
+    tag: golang-1.20
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.16"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: eco-ci-cd
```

Comment:
Could you please give a unique name for this instead of `eco-ci-cd`? I think it might be easier for us when we need to add notifications
This should be the name of your job, what about `far-edge-vran-dast` or simply `install-operators` if that's the only thing you are doing here?

Reply:
at the moment it is wip it will install the ocp and operator and run the dast test on top of it .
eco-ci-cd was updated as you suggested.